src/Microsoft.Health.Dicom.Core/Features/Telemetry/InstanceMeter.cs

```diff
@@ -0,0 +1,30 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Diagnostics.Metrics;
+
+namespace Microsoft.Health.Dicom.Core.Features.Telemetry;
+public sealed class InstanceMeter : IDisposable
+{
+    private readonly Meter _meter;
+
+    public InstanceMeter()
+    {
+        _meter = new Meter("Microsoft.Health.Dicom.Core.Features.Telemetry.Instance", "1.0");
+        InstanceCount = _meter.CreateCounter<double>(nameof(InstanceCount));
+        TotalInstanceBytes = _meter.CreateHistogram<double>(nameof(TotalInstanceBytes));
```

Comment:
For here and all of our metrics, can we add more metadata, like units and a description?

Reply:
Done. Please check if the unit and description makes sense.